Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/audit/PatientDiscoveryDeferredResponseAuditLogger.java

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
There are multiple spacing issues in this file.  Do you have the NB formatter set to format all lines on save?  If not, can you change that setting, go through each file you changed, and save them again?